integration-cli/docker_cli_run_test.go

```diff
@@ -1885,6 +1885,15 @@ func (s *DockerSuite) TestRunBindMounts(c *check.C) {
 		testRequires(c, DaemonIsLinux, NotUserNamespace)
 	}
 
+	if testEnv.OSType == "windows" {
+		v, err := kernel.GetKernelVersion()
```

Comment:
Should se use `system.GetOSVersion()` ? (or perhaps add it to `testEnv`, but that's a bit out of scope for this PR)

https://github.com/moby/moby/blob/dd945557876ba90c1b2c2eb6af17daf1dcdc83e3/daemon/daemon_windows.go#L199-L206

Reply:
I think someone needs to a sweep. The above pattern is pervasive through several tests. (Trying to do minimum needed to get CI back running and passing again - still heads down on this containerd work)

Reply:
@jhowardmsft why not just add `testRequires(c, DaemonIsWindowsAtLeastBuild(17763))` ?